.github/workflows/makefile.yml

```diff
@@ -0,0 +1,21 @@
+name: Makefile Checks
```

Comment:
Please rename this file to code format checks, also add a comment at the top explaining why we need a linux job in addition to the Windows job

Reply:
Done! Force-pushed to squash.